.eslintrc

```diff
@@ -25,6 +25,7 @@
           "**/*.config.{js,ts,mjs}",
           "**/*.setup.{js,ts}",
           ".storybook/**",
+          ".storybook/*.{ts,js}",
```

Comment:
Why is this needed? I'd expect the line above to lint all files under .storybook/. If they're not, then the line above should probably be fixed or removed.

Reply:
I changed this a while back to fix something that was happening but can't remember what exactly. it's likely merging the two would be what we want, e.g., `".storybook/**/*"` or something. I can test that out to confirm